db/README.md

```diff
@@ -0,0 +1,11 @@
+# db
+
+This directory contains database configuration, including and especially the database migrations.
+
+Run the migrations with [flyway](https://documentation.red-gate.com/flyway/flyway-cli-and-api/welcome-to-flyway), which can be installed with Homebrew.
+
+This directory includes config for connecting to the local database, and is run as part of the start script.
+
+`flyway migrate`
```

Comment:
I'm not quite clear about this -- is it saying that we should run `flyway migrate` directly? If so, under what circumstances?